src/logic/calculate.js

```diff
@@ -0,0 +1,129 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] After performing an operation and press `AC` it can be good if zero appears on the screen as default. please work on that. Currently, I can see a blank screen after pressing `AC` button.

Reply:
Hello,  @Dipeshtwis thank you for your comment. I already changed the code as you can see in the image below.
![tse requet](https://user-images.githubusercontent.com/60292761/133486357-45a64f23-72eb-4e71-a6ef-422e7d760fd8.PNG)